com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -12,8 +14,11 @@ namespace Unity.Netcode
     /// </summary>
     [AddComponentMenu("Netcode/" + nameof(NetworkObject), -99)]
     [DisallowMultipleComponent]
+
     public sealed class NetworkObject : MonoBehaviour
     {
+        public List<InterestNode> InterestNodes = new List<InterestNode>();
```

Comment:
Did you intend for this to show up in the inspector? Is this configuration or state? If it's state, should apply [NonSerialized] so that it doesn't appear in inspector or impact the serialized size of network object components